File: EquipmentApi/EquipmentApi/Entities/EquipmentPositionHistory.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace EquipmentApi.Entities
{
    public class EquipmentPositionHistory
    {
        public Guid EquipmentId { get; set; }
        public DateTime Date { get; set; }
        public float Lat { get; set; }
        public float Lon { get; set; }

        public Equipment Equipment { get; set; }
    }
}

File: project/Forest-operation-project/src/main/java/com/app/project/domain/EquipmentPositionHistory.java
package com.app.project.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.format.annotation.DateTimeFormat;

import javax.persistence.*;
import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Entity
@Table(name = "equipment_position_history")
public class EquipmentPositionHistory {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;

    @ManyToOne
    @JoinColumn(name = "equipment_id", referencedColumnName = "id")
    private Equipment equipment;

    private LocalDateTime date;

    private BigDecimal lat;

    private BigDecimal lon;
}
